Derivative regeneration: media-type option, refs #12903

replaceafill opened the pull request:
Added a command-line option, to the derivative regeneration task, that
filters based on the digital object media type ("audio" or "image" or
"text" or "video").

The confirmation message of the task has also been customized to this
effect.

sbreker commented on lib/task/digitalobject/digitalObjectRegenDerivativesTask.class.php:
```diff
@@ -35,6 +35,13 @@ protected function configure()
+        sfCommandOption::PARAMETER_OPTIONAL,
+        'Limit regenerating derivatives to a specific media type (e.g. "audio" or "image" or "text" or "video). "Other" is not supported',
+        null
+      ),
```
I would say this is ok to run all on one line. I think it'd be good to have consistency in the formatting of these rows so that this new one matches the others in terms of formatting.

replaceafill replied:
:+1:  Addressed in https://github.com/artefactual/atom/pull/895/commits/f39f50030a35e12370d781a5eb8998c71920b6dc

sbreker commented on lib/task/digitalobject/digitalObjectRegenDerivativesTask.class.php:
```diff
@@ -127,14 +150,16 @@ protected function execute($arguments = array(), $options = array())
       else
       {
-        $confirm[] = 'Continuing will regenerate the dervivatives for ALL digital objects';
+        $confirm[] = 'Continuing will regenerate the derivatives for '.$changed.' digital objects';
```
Nice catch! ;)

sbreker reviewed (approved):
Looks good @replaceafill!  Thanks!

replaceafill committed:
Derivative regeneration: media-type option, refs #12903

Added a command-line option, to the derivative regeneration task, that
filters based on the digital object media type ("audio" or "image" or
"text" or "video").

The confirmation message of the task has also been customized to this
effect.
